pyvista/utilities/misc.py

```diff
@@ -17,3 +18,123 @@ class PyvistaDeprecationWarning(Warning):
     """Non-supressed Depreciation Warning."""
 
     pass
+
+
+class VTKVersionInfo():
+    """Contains the VTK version information.
+
+    Wraps vtk.vtkVersion in a ``sys.version_info`` like manner.
+
+    Notes
+    -----
+    The micro attribute may be either the build date in YYYYMMDD or
+    the patch release.  Use caution when comparing version patch
+    release.
+
+    Examples
+    --------
+    Simply output the vtk version.
+
+    >>> import pyvista
+    >>> pyvista.vtk_version_info  # doctest:+SKIP
+    VTK Version 9.1.0
+
+    Compare the VTK version.
+
+    >>> pyvista.vtk_version_info > (9, 0)  # doctest:+SKIP
+    True
+
+    Return the major version of vtk.
+
+    >>> pyvista.vtk_version_info.major  # doctest:+SKIP
+    9
+    """
+
+    def __init__(self):
+        """Initialize the vtk version from the VTK class."""
+        # Get the VTK version. The API here is consistent since at least VTK v4.0.0
+        try:
+            ver = _vtk.vtkVersion()
+            self._major = ver.GetVTKMajorVersion()
+            self._minor = ver.GetVTKMinorVersion()
+            self._micro = ver.GetVTKBuildVersion()
+        except:  # pragma: no cover
+            self._major, self._minor, self._micro = (4, 0, 0)
```

Comment:
I think a warning here makes sense.  PyVista doesn't even support v4.0.0 of vtk, so it will be confusing if someone ever finds the version set to `(4, 0, 0)` without a warning.

Reply:
Added a warning in 58e7aa18bb212fcbd21b9f225bf7c4cb86ae3a19.